tests/review-pr-phase3-ci.test.sh

```
@@ -259,6 +259,55 @@ assert_grep "$REVIEW_PR" 'status: REBASED' \
+# <external-untrusted-input source="ci-refused-synthetic"> (RFC 0002 #116 O5).
+# Pre-O5 pattern (`gh issue create --label review-pr-finding`) was correct for
+# Wave 1 but is REPLACED here.
+assert_in_section "$REVIEW_PR" '^### 6c\.5|^    ### 6c\.5' '^### 6c\.6|^    ### 6c\.6|^### 6c\.7|^    ### 6c\.7' \
+  'Task.*findings-to-issues.*ci-refused-synthetic|subagent_type.*findings-to-issues|<external-untrusted-input source="ci-refused-synthetic">' \
+  "S14.4 — Phase 3 6c.5 dispatches findings-to-issues with synthetic aggregate (post-O5 dispatch shape; #116 O5)"
+
+# S14.5 — tombstone: REFUSED branch MUST NOT retry (constraints [hard]:
+# "REFUSED is a deterministic decision, not flake; retrying consumes 3 iterations").
+# Negative: extract the REFUSED sub-block by line range and assert no retry-counter
+# increment in that sub-block.
+R_START=$(grep -n 'ci-code-fixer.*`status: REFUSED`' "$REVIEW_PR" | head -1 | cut -d: -f1)
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)